device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C72/0/j2p-a7800r3a-36d-36x400G.config.bcm

```diff
@@ -994,3 +994,6 @@ serdes_tx_taps_33=nrz:-5:83:-22:0:0:0
 serdes_tx_taps_34=nrz:-5:83:-22:0:0:0
 serdes_tx_taps_35=nrz:-4:75:-21:0:0:0
 serdes_tx_taps_36=nrz:-8:89:-29:0:0:0
+
+xflow_macsec_secure_chan_to_num_secure_assoc_encrypt=2
+xflow_macsec_secure_chan_to_num_secure_assoc_decrypt=2
```

Comment:
The reason we chose to use 2 is because on J2C+ the max_sa_per_sc is 2 (we'll never use ANs 2 or 3) so we can't utilize all 4 SAs on the decrypt side.
So in result using decrypt=4 will just burn half the decrypt table scale.
However for macsec the scale will be limited by the number of ports so I don't think we can even reach half the decrypt scale.
So I think it's fine to use decrypt=4 to be consistent with Broadcom's recommendations.